python/hsfs/core/vector_server.py

```diff
@@ -301,7 +330,7 @@ def get_feature_vectors(self, entry, return_type, passed_features=[]):
             return np.array(vectors)
         elif return_type.lower() == "pandas":
             pandas_df = pd.DataFrame(vectors)
-            pandas_df.columns = feature_names
+            pandas_df.columns = self._feature_vector_col
```

Comment:
The same here, column names are strange:
<html><head></head><body>

  | Training Dataset Feature('id1', 'bigint', 0, False, None, None, 14) | Training Dataset Feature('fg1_id1', 'bigint', 1, False, None, None, 15) | Training Dataset Feature('fg1_id2', 'bigint', 2, False, None, None, 15)
-- | -- | -- | --
7 | 7 | 8
6 | 6 | 7

</body></html>